app/assets/stylesheets/vendor/main.css

```diff
@@ -4192,6 +4202,7 @@ aside { height: 100%; }
 
 .map article.widthS, .timeContainer.widthS {
     width: 100%;
+    margin: 0;
```

Comment:
This CSS is visible in mobile view.  It overrides the negative margin set for the desktop view.